README.md

```diff
@@ -105,7 +105,9 @@ only when a rule or set of rules explicitly or implicitly permits it.
 Mutable Rules 
 -------------
 
-**201** Players may take their turn at any time, independent of other players. 
+**303** Players may take a turn at any time, independent of other players.
+Players are allowed to take any number of turns at the same time, regardless
+of any other ongoing turns taken by them.
 All players begin with zero points.
```

Comment:
I put that part in to explictly allow ``asynchronous'' rule-change proposals, in addition to opening multiple PRs at the same time. I'm not sure it's necessary, but explicit is better than implicit?